pkijs/pkijs-tests.ts

```diff
@@ -1,17 +1,17 @@
 import * as asn1js from "asn1js";
 import { arrayBufferToString, stringToArrayBuffer, bufferToHexCodes } from "pvutils";
-import { getCrypto, getAlgorithmParameters } from "pkijs/src/common";
-import Certificate from "pkijs/src/Certificate";
-import CertificateRevocationList from "pkijs/src/CertificateRevocationList";
-import AttributeTypeAndValue from "pkijs/src/AttributeTypeAndValue";
-import Extension from "pkijs/src/Extension";
-import Attribute from "pkijs/src/Attribute";
-import SignedData from "pkijs/src/SignedData";
-import EncapsulatedContentInfo from "pkijs/src/EncapsulatedContentInfo";
-import SignerInfo from "pkijs/src/SignerInfo";
-import IssuerAndSerialNumber from "pkijs/src/IssuerAndSerialNumber";
-import SignedAndUnsignedAttributes from "pkijs/src/SignedAndUnsignedAttributes";
-import ContentInfo from "pkijs/src/ContentInfo";
+import { getCrypto, getAlgorithmParameters } from "pkijs";
+import { Certificate } from "pkijs";
```

Comment:
I know about it. Thank you.

Some costomers want to use `rollup` for ES2015 (multi entrypoints). I'll think about it for next update